sdk/tests/js/webgl-test-harness.js

```diff
@@ -238,21 +238,16 @@ var getMajorVersion = function(versionString) {
 
 /**
  * @param {string} url Base URL of the test.
- * @param {number} webglVersion Integer containing the WebGL major version.
- * @param {boolean} dumpShaders add dumpShader query parameter if true.
+ * @param {map} options Map of options to append to the URL's query string.
```

Comment:
Rather, the ones that are supported. It's clear that the options map is explicitly queried inside this function.